specification/sql/resource-manager/Microsoft.Sql/preview/2021-02-01-preview/examples/LedgerDigestUploadDisable.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "ledgertestrg",
+    "serverName": "ledgertestserver",
+    "databaseName": "testdb",
+    "ledgerName": "current",
+    "api-version": "2021-02-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/ledgertestrg/providers/Microsoft.Sql/servers/ledgertestserver/databases/testdb/current/disable",
+        "name": "current",
+        "type": "Microsoft.Sql/servers/databases/ledgerDigestUpload",
+        "properties": {
+          "state": "Disabled"
+        }
+      }
+    },
+    "202": {}
```

Comment:
>  "202": {} [](start = 2, length = 11)

202 response should include a location header and a URI - [https://github.com/Azure/azure-resource-manager-rpc/blob/2e1f2c11ce4cfdf309f9cc8ae71575cb8fb83319/v1.0/Addendum.md#202-accepted-and-location-headers](https://github.com/Azure/azure-resource-manager-rpc/blob/2e1f2c11ce4cfdf309f9cc8ae71575cb8fb83319/v1.0/Addendum.md#202-accepted-and-location-headers)

Reply:
Can you give an example of another response that has this? I am not able to find any for any other proxy resources or even CreateOrUpdate databases

Reply:
Here's an example: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/Microsoft.Resources/stable/2020-10-01/examples/PostDeploymentWhatIfOnResourceGroup.json